v2/components/APYTooltip.tsx

```diff
@@ -0,0 +1,54 @@
+import { FC, forwardRef } from "react";
```

Comment:
@larrythecucumber321 there's already a `MoreInfo` component that is exactly the same as this. Looking at it now the name isn't great. We should have a reusable component for all tooltips.

Reply:
Thanks for making me aware of this, I've modified `MoreInfo` to be able to accomodate another text input and deleted `APYToolTip`.